internal/dag/httpproxy_processor.go

```diff
@@ -1191,6 +1153,152 @@ func (p *HTTPProxyProcessor) rootAllowed(namespace string) bool {
 	return false
 }
 
+func (p *HTTPProxyProcessor) computeVirtualHostAuthorization(validCond *contour_api_v1.DetailedCondition, httpproxy *contour_api_v1.HTTPProxy) *GlobalExternalAuthorization {
+	if p.GlobalExternalAuthorization != nil && !httpproxy.Spec.VirtualHost.DisableGlobalAuthorization() {
```

Comment:
Per previous suggestions:
```suggestion
	if p.GlobalExternalAuthorization != nil && !httpproxy.Spec.VirtualHost.DisableAuthorization() {
```